pkg/network/tracer/tracer.go

```diff
@@ -19,6 +19,8 @@ import (
 	"github.com/cilium/ebpf"
 	"go.uber.org/atomic"
 
+	"github.com/DataDog/ebpf-manager/tracefs"
```

Comment:
should be grouped with above external packages: `goimports -local github.com/DataDog/datadog-agent` should do the right thing.